Diddy Kong Changes

Wish325 opened the pull request:
Been Working on these for a little bit. This is the first draft of changes for the character. I will be updating this in the coming days. But please test this and let me know your thoughts.

Change Log: 
[Diddy Kong Changes.txt](https://github.com/HDR-Development/HewDraw-Remix/files/12065908/Diddy.Kong.Changes.txt)

Assets: 
[fighter.zip](https://github.com/HDR-Development/HewDraw-Remix/files/12065932/fighter.zip)

Wish325 committed:
Diddy Kong Changes

Been Working on these for a little bit. This is the first draft of changes for the character. I will be updating this in the coming days. But please test this and let me know your thoughts.

jobrien97 committed:
Merge branch 'dev' into pr/1806

jobrien97 committed:
remove recoil from popgun fire

jobrien97 committed:
oopsie

jobrien97 committed:
grounded upB angleability

jobrien97 committed:
revert special_hi_charge_ang_f_max

jobrien97 committed:
dash attack adjustment

hdr-balance-backroom reviewed (dismissed):
- Add landing lag to dash attack slideoff
- Reduce the window to input the kick during Monkey Flip
- Increase Monkey Flip FAF

Wish325 committed:
Merge remote-tracking branch 'upstream/dev' into Diddy-Kong

Wish325 committed:
Its done, i've finished

Diddy side b is no longer a command grab. it is now a movement option actionable frame 26

Dash attack is now able to fastfall the entire duration of the move and has 7 frames of landing lag

Up air and dash attack now have proper effect trails.

SkewedAskew committed:
Merge branch 'dev' into pr/1806

blu-dev committed:
remove remaining dash attack fastfall

Wish325 committed:
Update opff.rs

Wish325 committed:
done

SkewedAskew committed:
Update opff.rs

SkewedAskew committed:
Small cleanup

SkewedAskew committed:
Motion rate cleanup
